cursorless-talon/src/spoken_scope_forms.py

```diff
@@ -0,0 +1,76 @@
+from talon import Context
+
+from .csv_overrides import csv_get_ctx, csv_get_normalized_ctx
+
+
+def init_scope_spoken_forms(graphemes_talon_list: dict[str, str]):
+    create_flattened_talon_list(csv_get_ctx(), graphemes_talon_list, True)
+    create_flattened_talon_list(csv_get_normalized_ctx(), graphemes_talon_list, False)
+
+
+def create_flattened_talon_list(
+    ctx: Context, graphemes_talon_list: dict[str, str], include_custom_regex: bool
+):
+    scope_types_singular: dict[str, str] = {}
+    scope_types_plural: dict[str, str] = {}
+
+    append_talon_lists(
+        ctx, scope_types_singular, scope_types_plural, "simple", "cursorless_scope_type"
+    )
+    append_talon_lists(
+        ctx,
+        scope_types_singular,
+        scope_types_plural,
+        "surroundingPair",
+        "cursorless_surrounding_pair_scope_type",
+    )
+    append_talon_lists(
+        ctx,
+        scope_types_singular,
+        scope_types_plural,
```

Comment:
You can reduce the duplication here by making a dict and iterating through it. It will also make the a little bit different further down more obvious

```
    x = {
        "simple": "cursorless_scope_type",
...
     }
```